app/views/docs/index.phtml

```diff
@@ -103,6 +110,7 @@ $cols = [
                     <li><a href="/docs/keys">API Keys</a></li>
                     <li><a href="/docs/permissions">Permissions</a></li>
                     <li><a href="/docs/events">Events</a></li>
+                    <li><a href="/docs/queries">Queries</a></li>
                     <li><a href="/docs/pagination">Pagination</a></li>
```

Comment:
We can remove these from here then ? If they are in the databases section already?

Reply:
@christyjacob4 @eldadfux Suggested doing these in multiple places for discoverability. 
1. It's default collapsed in DB
2. It also applies to APis outside DB

Reply:
Got it 👍